src/app/containers/MostRead/index.stories.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs } from '@storybook/addon-knobs';
+import { withServicesKnob } from '@bbc/psammead-storybook-helpers';
+import MostReadContainer from './Canonical';
+import { RequestContextProvider } from '#contexts/RequestContext';
+import { ServiceContextProvider } from '#contexts/ServiceContext';
+
+const staticMostReadURL = service => `/data/${service}/mostRead/index.json`;
+
+const renderMostReadContainer = (service, isAmp) => (
+  <RequestContextProvider
+    bbcOrigin={`/${service}/articles/c0000000000o`}
+    id="c0000000000o"
+    isAmp={isAmp}
+    pageType="article"
+    service={service}
+    statusCode={200}
+    pathname={`/${service}`}
+    variant={null}
```

Comment:
`RequestContextProvider` and `ServiceContextProvider` have `variant` default value of `null`. We don't need to explicitly set them again.

Reply:
good spot, I've updated this now so that we now get a variant value and pass it to these ContextProviders. but they aren't really dependant on it 🤔